ic-os/components/misc/config.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Shared config utilities.
+
+CONFIG_FILE="/var/ic/config/config.json"
```

Comment:
Consider moving `CONFIG_FILE` into get_config_value as a local otherwise it may get overwritten (it's also a pretty generic name).
Alternatively, you could mark it as readonly (`readonly CONFIG_FILE="..."`) but even then, if someone sources this file, there could be a name clash.

Reply:
Updated! I moved it to be a local variable.